src/Kestrel.Core/Internal/Http/Frame.Generated.cs

```diff
@@ -75,7 +75,7 @@ private void FastReset()
             _currentIHttpSendFileFeature = null;
         }
 
-        internal object FastFeatureGet(Type key)
+        public object FastFeatureGet(Type key)
```

Comment:
I think you need to regenerate files to update the (temporary) Http2Stream.Generated.cs file.

https://travis-ci.org/aspnet/KestrelHttpServer/jobs/265824703#L912-L928